tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,338 @@
+
+**Met**
+
+> The project MUST either:
+>   1. make it easy to identify and update reused externally-maintained components; or
+>   2. use the standard components provided by the system or programming language.
+> 
+> Then, if a vulnerability is found in a reused component, it will be easy to update that component.
+
+**Met**
+
+
+> The project SHOULD avoid using deprecated or obsolete functions and APIs where FLOSS alternatives are available in the set of technology it uses (its "technology stack") and to a supermajority of the users the project supports (so that users have ready access to the alternative).
+
+_Possible answers: Met/Unmet/NA_
+
+
+## Automated test suite
+
+> An automated test suite MUST be applied on each check-in to a shared repository for at least one branch. This test suite MUST produce a report on test success or failure.
+
+**Met**
+
+> The project MUST add regression tests to an automated test suite for at least 50% of the bugs fixed within the last six months.
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST have FLOSS automated test suite(s) that provide at least 80% statement coverage if there is at least one FLOSS tool that can measure this criterion in the selected language.
+
+_Possible answers: Met/Unmet/NA_
+
+
+## New functionality testing
+
+> The project MUST have a formal written policy that as major new functionality is added, tests for the new functionality MUST be added to an automated test suite.
+
+**Met**
+
+> The project MUST include, in its documented instructions for change proposals, the policy that tests are to be added for major new functionality.
+
+**Met**
+
+## Warning flags
+
+> Projects MUST be maximally strict with warnings in the software produced by the project, where practical.
+
+_Possible answers: Met/Unmet/NA_
+
+
+# Security
+
+## Secure development knowledge
+
+> The project MUST implement secure design principles (from "know_secure_design"), where applicable. If the project is not producing software, select "not applicable" (N/A).
+
+** Met**
+
+
+## Use basic good cryptographic practices
+
+_Note that some software does not need to use cryptographic mechanisms. If your project produces software that (1) includes, activates, or enables encryption functionality, and (2) might be released from the United States (US) to outside the US or to a non-US-citizen, you may be legally required to take a few extra steps. Typically this just involves sending an email. For more information, see the encryption section of [Understanding Open Source Technology & US Export Controls](https://www.linuxfoundation.org/resources/publications/understanding-us-export-controls-with-os-projects/)._
+
+> The default security mechanisms within the software produced by the project MUST NOT depend on cryptographic algorithms or modes with known serious weaknesses (e.g., the SHA-1 cryptographic hash algorithm or the CBC mode in SSH).
+
+_Possible answers: Met/Unmet/NA_
```

Comment:
I think is met, but a lot depends on how the final users will build the applications (ex. md5 to store a pass, etc..). Maybe is `N/A`?

Reply:
@tniessen any thoughts?

Reply:
I think we provide those algorithms to users, but dont rely on them

Reply:
Confirmed, this is met. We don't use weak algorithms internally.

Reply:
resolved in 80520539e11cc910ef838962808e7023f514f7f6